lib/Doctrine/DBAL/Driver/Mysqli/MysqliStatement.php

```diff
@@ -97,14 +102,14 @@ public function __construct(mysqli $conn, $prepareString)
      */
     public function bindParam($column, &$variable, $type = ParameterType::STRING, $length = null)
     {
+        assert(is_int($column));
```

Comment:
Wondering if we should run CI with `dzend_assertions` enabled and throwing

Reply:
According to the [documentation](http://php.net/manual/en/function.assert.php#function.assert.expectations), `zend.assertions` are enabled by default (development mode) and according to the internet, Travis CI doesn't disable them. As for `assert.exception` regardless of the configuration, PHPUnit will handle a warning or a `Throwable` and turn into an error. So unless we want to be paranoid, the existing configuration is good enough.

Reply:
:+1: